tests/gtest-nano.h

```diff
@@ -0,0 +1,66 @@
+// gtest-nano.h implements very minimalistic GTest-compatible API that can be used to run tests in older
+// (C++98-compatible) environments.
+
+#include <iostream>
+#include <vector>
+
+namespace testing {
+    struct TestInfo {
+        const char* suite;
```

Comment:
Generally, I strongly recommend moving the `*` (for pointers) and `&` (for references) closer to the variable name because of how the C/C++ semantics work when declaring multiple variables (this is also the reason I took the liberty of reformatting the existing code with that in mind in commit https://github.com/kaitai-io/kaitai_struct_cpp_stl_runtime/commit/aaa2a3ff7ce5981d9ab1f3ada36e6633c5e96162).

```diff
-        const char* suite;
+        const char *suite;
```

This convention allows you to correctly predict what will be the result of the following code snippets:

```cpp
int foo(void) {
    int *a = 0, b = 0;
    return a == 0 ? a : b;
}
// test.cpp:3:19: error: incompatible operand types ('int *' and 'int')
//     return a == 0 ? a : b;
//                   ^ ~   ~
```

(☝️ And note that `clang` also prefers to print the type as `'int *'` in the error message, in line with my preferences.)

```cpp
#include <iostream>
int main() {
    std::string s = "Changed";
    std::string &x = s, y = s;

    x += " through x";
    y += " through y";

    std::cout << s << std::endl; // => "Changed through x"
}
```

I mean, we're just humans, so code formatting sometimes plays a big role how we may (mis)interpret things, and if you write <code><del>int*</del> a = 0, b = 0</code> (or <code><del>std::string&</del> x = s, y = s</code>) instead (which I don't recommend), it's easy to get fooled and think that both `a`, `b` have the type <code><del>int*</del></code> (which is not the case) etc.

Reply:
I've missed the change, thanks for pointing out. At the very least this should be consistent with the rest of the codebase.

We can have separate discussion on pros and cons of both of these approaches. From my personal experience (although I'm biased, as a human being), in the latest years `int* a` is a more popular one than `int *a`, and double declaration on the same command line is normally banned in the style guide and checker.

Reply:
I believe it's done, please take one final look?